src/components/Layout/index.jsx

```diff
@@ -72,11 +72,11 @@ export default class Layout extends React.Component {
       backBtn,
       hasSearch,
       title,
-      noLogin
```

Comment:
this flag will be removed, no need to use

Reply:
ok.